packages/mask/src/plugins/EVM/UI/Web3State/getAssetsFn.ts

```diff
@@ -11,15 +11,35 @@ import {
     Web3ProviderType,
     FungibleAssetProvider,
     createExternalProvider,
+    getCoinGeckoCoinId,
+    CurrencyType,
+    CryptoPrice,
+    PriceRecord,
+    ChainId,
 } from '@masknet/web3-shared-evm'
+import BigNumber from 'bignumber.js'
 import { Pageable, Pagination, TokenType, Web3Plugin } from '@masknet/plugin-infra'
 import BalanceCheckerABI from '@masknet/web3-contracts/abis/BalanceChecker.json'
 import ERC721ABI from '@masknet/web3-contracts/abis/ERC721.json'
 import type { AbiItem } from 'web3-utils'
 import { uniqBy } from 'lodash-unified'
 import { PLUGIN_NETWORKS } from '../../constants'
 import { makeSortAssertWithoutChainFn } from '../../utils/token'
+import { createGetLatestBalance } from './createGetLatestBalance'
 import type { ERC721 } from '@masknet/web3-contracts/types/ERC721'
+import { pow10 } from '@masknet/web3-shared-base'
+
+// tokens unavailable neither from api or balance checker.
+// https://forum.conflux.fun/t/how-to-upvote-debank-proposal-for-conflux-espace-integration/13935
+const TokenUnavailableFromDebankList = [ChainId.Conflux]
+
+const TOKEN_PRICE_COINGECKO_URL_BASE = 'https://api.coingecko.com/api/v3'
+
+async function getNativeTokenPrice(tokenId: string) {
+    const requestPath = `${TOKEN_PRICE_COINGECKO_URL_BASE}/simple/price?ids=${tokenId}&vs_currencies=${CurrencyType.USD}`
+    const prices = await fetch(requestPath).then((r) => r.json() as Promise<CryptoPrice>)
+    return prices
+}
```

Comment:
Recommend to use the [TokenPriceAPI](packages/web3-providers/src/token-price/index.ts).

Reply:
done